frontend/src/metabase/components/CodeBlock/utils.ts

```diff
@@ -0,0 +1,119 @@
+import { html } from "@codemirror/lang-html";
+import { javascript } from "@codemirror/lang-javascript";
+import { json } from "@codemirror/lang-json";
+import { python } from "@codemirror/lang-python";
+import { StreamLanguage } from "@codemirror/language";
+import { clojure } from "@codemirror/legacy-modes/mode/clojure";
+import { pug } from "@codemirror/legacy-modes/mode/pug";
+import { ruby } from "@codemirror/legacy-modes/mode/ruby";
+import type { EditorState, Extension } from "@codemirror/state";
+import {
+  Decoration,
+  EditorView,
+  RangeSet,
+  type ReactCodeMirrorRef,
+  StateEffect,
+  StateField,
+} from "@uiw/react-codemirror";
+import { handlebarsLanguage as handlebars } from "@xiechao/codemirror-lang-handlebars";
+import { getNonce } from "get-nonce";
+import { type RefObject, useEffect } from "react";
+
+import S from "./CodeBlock.module.css";
+import type { CodeLanguage } from "./types";
+
+export function getLanguageExtension(language: CodeLanguage): Extension {
+  switch (language) {
+    case "clojure":
+      return StreamLanguage.define(clojure);
+    case "erb":
+    case "html":
+      return html();
+    case "json":
+      return json();
+    case "python":
+      return python();
+    case "mustache":
+      return handlebars;
+    case "jade":
+    case "pug":
+      return StreamLanguage.define(pug);
+    case "ruby":
+      return StreamLanguage.define(ruby);
+    case "javascript":
```

Comment:
where do we define `javascript`? I thought we only use `typescript`?https://github.com/metabase/metabase/blob/cc016e3dd4ad3c0a2532d268ca90da787bb02ad9/frontend/src/metabase/public/lib/code.ts#L40

Reply:
This is just there for completeness.

Reply:
just looks like a misdirection rather than for completeness. When I see this, I'd expect to be able to select both JS and TS somewhere in our app, but if that's not the case I'd rather have one. What do you think?

Reply:
I'd personally remove `javascript` here with the reason mentioning above what do you think?